deezy.js

```diff
@@ -11,7 +12,12 @@ async function post_scan_request(request_body) {
     check_api_key()
     const url = `${BASE_URL}/sat-hunting/scan`
     const { data } = await axios
-        .post(url, request_body, { headers: { 'x-api-token': process.env.DEEZY_API_KEY } })
+        .post(url, request_body, {
+            headers: {
+                'x-api-token': process.env.DEEZY_API_KEY,
+                'User-Agent': `sat-hunter/${packageInfo.version}`, // Include the app version
```

Comment:
what about extracting it as variable and re-use in all methods in this file?

Reply:
even better would be usage of axios interceptor, 
in this case ALL requests will include this header without need of us explicitly including them on every call

Reply:
I agree

Reply:
will do, good idea